app/views/reviews/_form.html.erb

```diff
@@ -0,0 +1,15 @@
+<h3>Post a Review</h3>
+
+<%= form_for [@book, @review] do |f| %>
+  <p>
+    <%= f.label :body %><br/>
+    <%= f.text_area :body %>
+  </p>
+  <p>
+    <%= f.label :rating %><br/>
+    <%= f.text_area :rating %>
+  </p>
+  <p>
+    <%= f.submit 'Submit' %>
+  </p>
+<% end %>
```

Comment:
None of the user stories ask for a review to be created. You don't need this form.